cmd/sealer/cmd/cluster/apply.go

```diff
@@ -142,67 +145,103 @@ func createNewCluster(clusterImageName string, infraDriver infradriver.InfraDriv
 		clusterLaunchCmds = infraDriver.GetClusterLaunchCmds()
 	)
 
-	clusterHostsPlatform, err := infraDriver.GetHostsPlatform(clusterHosts)
-	if err != nil {
-		return err
-	}
+	if applyMode == common.ApplyModeLoadImage {
+		logrus.Infof("start to apply with mode(%s)", applyMode)
 
-	imageMounter, err := imagedistributor.NewImageMounter(imageEngine, clusterHostsPlatform)
-	if err != nil {
-		return err
-	}
+		clusterHostsPlatform, err := infraDriver.GetHostsPlatform(clusterHosts)
+		if err != nil {
+			return err
+		}
 
-	imageMountInfo, err := imageMounter.Mount(clusterImageName)
-	if err != nil {
-		return err
-	}
+		imageMounter, err := imagedistributor.NewImageMounter(imageEngine, clusterHostsPlatform)
+		if err != nil {
+			return err
+		}
 
-	defer func() {
-		err = imageMounter.Umount(imageMountInfo)
+		imageMountInfo, err := imageMounter.Mount(clusterImageName)
 		if err != nil {
-			logrus.Errorf("failed to umount cluster image")
+			return err
 		}
-	}()
 
-	distributor, err := imagedistributor.NewScpDistributor(imageMountInfo, infraDriver, cf.GetConfigs())
-	if err != nil {
-		return err
-	}
+		distributor, err := imagedistributor.NewScpDistributor(imageMountInfo, infraDriver, cf.GetConfigs())
+		if err != nil {
+			return err
+		}
 
-	plugins, err := loadPluginsFromImage(imageMountInfo)
-	if err != nil {
-		return err
-	}
+		if err := distributor.DistributeRegistry(infraDriver.GetHostIPList()[0], infraDriver.GetClusterRootfsPath()); err != nil {
```

Comment:
if applyMode == common.ApplyModeLoadImage {
if err := distributor.DistributeRegistry(infraDriver.GetHostIPList()[0], infraDriver.GetClusterRootfsPath());err!=nil{}

return nil 

}

Reply:
Is it possible to implement the loadImage apply mode by the above few lines of code?